components/tools/FiltersDropdown.js

```diff
@@ -12,18 +12,18 @@ export default function FiltersDropdown({ dataList = [], checkedOptions = [], se
   };
 
   return (
-    <div className={twMerge(`max-w-lg flex gap-2 flex-wrap p-2 duration-200 delay-150 ${className}`)} data-testid="FiltersDropdown-div">
+    <div className={twMerge(`max-w-lg max-h-[25rem] overflow-y-auto border-4 flex gap-2 flex-col p-2 duration-200 delay-150 ${className}`)} data-testid="FiltersDropdown-div">
```

Comment:
I still see that changes I requested before are reverted in the latest change. Kindly update the PR.